polynote-kernel/src/main/scala/polynote/kernel/PolyKernel.scala

```diff
@@ -173,6 +176,16 @@ class PolyKernel private[kernel] (
                       IO.pure(result)
                   }
                 }
+
+                for {
+                  _ <- symbolTable.drain()
+                  start <- clock.monotonic(MILLISECONDS)
+                  res <- results
+                  end <- clock.monotonic(MILLISECONDS)
+                  timestamp <- clock.realTime(MILLISECONDS)
```

Comment:
Should the timestamp be when the execution *started*? TBH I'm not sure what people would expect (or if they care one way or the other)

Reply:
That's a good question. I just tried on Zep and it looks like the end time is used there.